### PR TITLE
CNS 438 - refactor tests to use subscription

### PR DESCRIPTION
**Pull request by Yaroms**

### CODE REVIEW

File: x/pairing/keeper/msg_server_relay_payment_test.go [hunk 1]
```diff
@@ -810,14 +735,11 @@ func TestCuUsageInProjectsAndSubscription(t *testing.T) {
...
 	projKeeper := ts.keepers.Projects
 
-	err := ts.addClient(1)
-	require.Nil(t, err)
-
-	subscriptionOwner := ts.providers[0].Addr.String()
-	projectAdmin1 := ts.clients[0].Addr.String()
-	projectAdmin2 := ts.clients[1].Addr.String()
+	subscriptionOwner := ts.clients[0].Addr.String()
+	proj1PK, projectAdmin1 := sigs.GenerateFloatingKey()
```
Comment (oren-lava) [inline x/pairing/keeper/msg_server_relay_payment_test.go:739]: should proj1PK be renamed to proj1SK? (PK is usually public key)

**Review by oren-lava — DISMISSED** [review]

File: x/pairing/keeper/msg_server_relay_payment_gov_test.go [hunk 2]
```diff
@@ -958,12 +751,11 @@ func verifyRelayPaymentObjects(t *testing.T, ts *testStruct, relayRequest *pairi
...
-	clientStakeEntry, err := ts.keepers.Epochstorage.GetStakeEntryForClientEpoch(sdk.UnwrapSDKContext(ts.ctx), relayRequest.GetSpecId(), ts.clients[0].Addr, uint64(relayRequest.GetEpoch()))
+	qresponse, err := ts.keepers.Pairing.UserEntry(ts.ctx, &pairingtypes.QueryUserEntryRequest{Address: ts.clients[0].Addr.String(), ChainID: relayRequest.GetSpecId(), Block: uint64(relayRequest.GetEpoch())})
 	require.Nil(t, err)
-	clientMaxCU, err := ts.keepers.Pairing.ClientMaxCUProviderForBlock(sdk.UnwrapSDKContext(ts.ctx), uint64(relayRequest.GetEpoch()), clientStakeEntry)
-	require.Nil(t, err)
-	if clientMaxCU < relayRequest.CuSum {
-		require.Equal(t, relayRequest.GetCuSum(), clientMaxCU)
+
+	if qresponse.MaxCU < relayRequest.CuSum {
+		require.Equal(t, relayRequest.GetCuSum(), qresponse.MaxCU)
```
[the remark this answers is not in the document]
Comment (Yaroms) [reply x/pairing/keeper/msg_server_relay_payment_gov_test.go:758]: you wrote this test :) i dont remember

[the remark this answers is not in the document]
Comment (oren-lava) [reply x/pairing/keeper/msg_server_relay_payment_gov_test.go:758]: the checks are unecessary, so I've removed them

File: x/pairing/keeper/msg_server_relay_payment_gov_test.go [hunk 3]
```diff
@@ -978,8 +770,8 @@ func verifyRelayPaymentObjects(t *testing.T, ts *testStruct, relayRequest *pairi
 	require.Equal(t, true, found)
 	require.Equal(t, uint64(relayRequest.GetEpoch()), uniquePaymentStorageClientProvider.GetBlock())
 
-	if clientMaxCU < relayRequest.CuSum {
-		require.Equal(t, relayRequest.GetCuSum(), clientMaxCU)
+	if qresponse.MaxCU < relayRequest.CuSum {
+		require.Equal(t, relayRequest.GetCuSum(), qresponse.MaxCU)
```
[the remark this answers is not in the document]
Comment (oren-lava) [reply x/pairing/keeper/msg_server_relay_payment_gov_test.go:774]: the checks are unecessary, so I've removed them

**Review by oren-lava — APPROVED** [review]

**Pull request merged**